docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+
+.NET Core 2.2 is supported on the following Linux distributions/versions:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                             |  Version                |  Architectures   |
+| ------------------------------ | ----------------------- | ---------------- |
+| Red Hat Enterprise Linux       |  6, 7                   | x64 |
+| CentOS                         |  7                      | x64 |
+| Oracle Linux                   |  7                      | x64 |
+| Fedora                         |  29, 30                 | x64 |
+| Debian                         |  9                      | x64, ARM32 |
+| Ubuntu                         |  16.04, 18.04, 18.10, 19.04    | x64, ARM32 |
+| Linux Mint                     |  17, 18                 | x64 |
+| openSUSE                       |  15+                    | x64 |
+| SUSE Enterprise Linux (SLES)   |  12 SP2+                | x64 |
+| Alpine Linux                   |  3.7+                   | x64 |
+
+For more information about .NET Core 2.2 supported OS, distributions, and lifecycle policy, see [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md).
+
+# [.NET Core 2.1](#tab/netcore21)
+
+.NET Core 2.1 treats Linux as a single operating system. There's a single Linux build (per chip architecture) for supported Linux distributions.
+
+.NET Core 2.1 is supported on the following Linux distributions/versions:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                             |  Version                |  Architectures   |
+| ------------------------------ | ----------------------- | ---------------- |
+| Red Hat Enterprise Linux       |  6, 7, 8                | x64 |
+| CentOS                         |  7+                     | x64 |
+| Oracle Linux                   |  7+                     | x64 |
+| Fedora                         |  29+                    | x64 |
+| Debian                         |  9                      | x64, ARM32 |
+| Ubuntu                         |  16.04, 18.04, 19.04, 19.10    | x64, ARM32 |
+| Linux Mint                     |  17+                    | x64 |
+| openSUSE                       |  15+                    | x64 |
+| SUSE Enterprise Linux (SLES)   |  12 SP2+                | x64 |
+| Alpine Linux                   |  3.7+                   | x64 |
+
+For more information about .NET Core 2.1 supported OS, distributions, and lifecycle policy, see [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md).
+
+---
+
+## Linux distribution dependencies
+
+Based on your linux distribution, you may need to install additional dependencies.
+
+> [!IMPORTANT]
+> The exact versions and names may vary slightly on your Linux distribution of choice.
+
+### Ubuntu
```

Comment:
This list is out of date.  
- libcurl is no longer a required library.  
- libssl is a dependency that .NET Core supports multiple versions of.
- etc.

I would suggest looking at the list the .NET Core debian packages have defined
- https://github.com/dotnet/core-setup/blob/master/src/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json
- https://github.com/dotnet/core-setup/blob/release/2.2/src/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json
- https://github.com/dotnet/core-setup/blob/release/2.1/src/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json

Reply:
@leecow What source should I use? the links above or the wiki pages you have from the .net core  repo?

Reply:
The current list was developed by dumping the dependencies expressed in the .deb and .rpm packages at the time. The sources Michael lists are a good start deciding how to update the list and/or we could look at how these are expressed in the final packages.